### DISCUSSION TITLE
Balance druid rotation

### DISCUSSION THREAD

**Discussion by bsvends1** [Hekili/hekili#884] [Q&A] [answered]
Hey guys!

After having Hekili helping getting my rotation in order on my rogue, I wanted to see if I could fine tune my casts, on my main - balance druid. However, using the rotation suggestions from the add-on, the dps drops about 50% compared to my standard rotation on most fights - long, short, boss, trash, etc. Tried several settings with more or less the same result. Simming my char, gives me around the same dps numbers that I get when using my  standard rotation. Any suggestions?

**Comment by Hekili** [comment, marked as answer]
There's really not enough information provided here to give suggestions.  This is more of a conversation to be had on Discord, with snapshots, logs, etc.

**Reply by bsvends1** [reply]
Ok, will do Discord, ty for the reply.
